### DISCUSSION TITLE
Tooltips for visNetwork

### DISCUSSION THREAD

**Discussion by franzbischoff** [ropensci/targets#366] [Ideas]
You can check the live test here: https://franzbischoff.github.io/false.alarm/

The proposal is, when you write `tar_visnetwork(TRUE, label = c("time", "size", "branches"))`, the "label" goes actually to the "title" field on visNetwork object. This is cleaner.

What do you think?

My hack:
```R
  network <- tar_visnetwork(TRUE, label = c("time", "size", "branches"))
  tips <- stringr::str_split_fixed(network$x$nodes$label, pattern = "\n", n = 2)[, 2]
  tips <- stringr::str_replace_all(tips, "\n", "<br />")
  network$x$nodes$title <- tips
  network$x$nodes$label <- network$x$nodes$name
```

**Comment by wlandau** [comment]
Tooltips are a great idea, but I do want to also keep the option of appended labels because it helps with `reprex`es and other situations where screenshots are necessary. And in general, there are a ton of features we could explore for graphs, and I would prefer to keep things simple for `tar_visnetwork()`.

`tar_network()` gives you the actual node and edge data frames, and you can create a `visNetwork` out of them pretty straightforwardly. This is the way I would recommend adding tooltips. Someone inspired enough might even write a separate package with a whole bunch of fancy `targets` graphs similar to how `ggthemes` extends `ggplot2`.

Here is how `tar_visnetwork()` internally uses the nodes and edge data frames you can get from `tar_network()`:

https://github.com/ropensci/targets/blob/0449d9cc96ae871032cfd1bfeeacb03eae87fd6d/R/class_visnetwork.R#L120-L145
